manifests/overlays/prod/secret.enc.yaml

```diff
@@ -5,38 +5,39 @@ metadata:
+    AUTH_GITHUB_CLIENT_ID: ENC[AES256_GCM,data:nsS34zUMw0Mxq8K8Me9UR+/R3KU=,iv:T+iGN2BrtW3rnmnsyPfm8Rm9bB/OwyHI6Y8I219I764=,tag:hAzjy2Mc6a+iExhs5lKgwQ==,type:str]
+    AUTH_GITHUB_CLIENT_SECRET: ENC[AES256_GCM,data:u2YvQSTTWQnWeaJ0i5SVNckLiypEwH06KlTkbnkCDzSLQ8U9Vvjxog==,iv:R3MgLJPTIktE1Dqmfig1R64n2T3u+0s8+/RdZONn2Hs=,tag:Hypw4miF0kmhwTq476PzNw==,type:str]
+    GRAFANA_TOKEN: ENC[AES256_GCM,data:K4BgARIFrGfaW6nZrru4UVN3DyLe13GN6wxnTAwAIezzWc4zcpUM7VpEV1vJHj6lNXuLHhZ4wXSFWWRNn/GZtLXh9iQ3V0VFuq/DJ5ZtjcKO5Jd1ygd9Ifqt4DM=,iv:MJusoT5JTdl8Hyy7dRPZeLdKSMlhRunQ/UHxd5+ABDU=,tag:VBNv8gjgG/QwcLuOIv7IlA==,type:str]
+    SERVICE_ACCOUNT_TOKEN: ENC[AES256_GCM,data:aY1OObUqxgTl759rhrBkQKwpXykmG8ssREzbbvg0TvZSQ1V6qyDyXyHpXAwQXbrm1TdE/7sfK4MlymYl5TYfNuoUP//6ycqedH0eO6rEwAkV699EgnAnsV4eBBcDnQk46X6JbChI3N1M/2c8f2zRN25ZJbz/TBkM8K1AI3M0Mj6LbPLyklpInQ50Uy7UM5VK3FDaTV4RFlt0H1wEhGSyPp/OdQMl7DQECZsDKc5RUtAPXlSg9Grx4+1ZcBtscaA9cB+U9IjUbfC2gR8k3Ea+rHK7DTON+KGFQn42eWOlWx6PKC1maA43qpLIXeOCDq/iABXv6zdNmrqp04uqPedCh18oXX3FwNXuJp6T9wGPB9G2u4ytFOqGWPkEe0a02aSoGgDnUJXDTHjOD3Dork/im6xw7DWGOGmIWYgvPfoUy/rkYvrpqn9Zrdsqthf1NGvjHtViEDsb7STHgc8b++HGBfs7+raibZDE3dogvjSdl9VIUK6yYYCQeJ7INXG8LtpDnmXuEQI4uQ0l1GCOXeQHaeVHhKwvNlYXufQvHQg3+mAXqzE1Q3jrASPO1fvOrCIQBJIUmPkIeGgz5dx/kEehVz5Y0W0224x9P6krnggojhBZFx4L32/ZEHbpbBCnRYm70mHw1oHu1cDf6jB5nIru4MQGkziWY6w6RxUfKXnyWQY3SOVk1QC6zy8IzWHmnZzYZZiZvxhmb4iq0SXhldHWhlS6AWzzrNDHsBKz09v7Tn0GIwsEhUn4GRFD/UHxehXfkcHjoVPVAo3UPadpcfNeyqkasKHsukB3FsqDE3/shOCYx9uEDQ67Ug4bd3NxgI5K1EQfyRacUkk8KSPVAc/bX8fNFdhaaC57TmPrX2pqoqwygEmxCWxCcn/dlOEcMCCiVhyALax6fHLzOU8HLB0m9jcvduhDU5WRw9dm4uCO2vIoSBPV7RoXAsL54elG4LHvT6gthi20TpRBEVwHJg1X+L41o8cqK9E3t0U/FzfYE8GYvoyFELOOLDYpX9ObZQtsbNfbUY86KE0Uo6fqhC8gLSBCKMLoofM5cywzQQWV5tUwFm0rD9UrKk5wgYohCBTp12oZO8YHnZ0CXIufDKR4iiRO43wjAv/ICHgThjz/U1rNUbtRrLWaD4jNlRAgRoMabaXoF0+iHcltMJZrPN6/TlpPJUdO+RENFFMS1S/YVEDeysqZ+0Pj+CGpef3pR8zg5HSEnt9nbXU6GY5SW0DX0mq7/HEQsKaO9pcNmfPsF3tGYTPMif+uO8vTfViZLFq4KN5GI/EPviPGUH5Yu1t99n+MWQjQqqedFA7/Vb3BtrZvHVlyYJWCgk6njDf7SZXfKnxg9dlZ5s8EiWeklFFLU0I0RXglTLS1jgUOo6PHZe8Y+LJjU6NJSR7jpqNYebBW5yjw7tD730KhSLbT8aIgW8Algjr33LwvaNDg7AatICaERLYBQyUZPN8RyvWPxjDHfv6L6h/jjWUtNQq8CKkT2q9sgB6J1dA8DQCp1bv0+ERxa5U2ch8+B496UAYgjojGubo3jtQyGl1pdwb9vUrV+zYMW1vPzkR0CR4AdPBwrjWayT1EIVAnQD3HoVNCi7IwsDvfj85n1w/abP4Z+DmSnJjoPUg+MtovICnAhPjTk/9BtlUDEYgiAcoLbG8wVx9Ady3osBikf9m9LmHytfcNu051ciu9/oepz6u+UPda/ErKBFlgVrI/75a2HJnmDN3xzrfQKea/coUTvNDauZ9FYcFBijyLCoRTbqTnj6+GY3iJ1kVaSa+XnEIL,iv:dRY2rW0ltUXcfZQsWoQqvaO+JsBbXIB+ATNe7jlcenk=,tag:sn+6//VN1DCCPWAqelo2Xg==,type:str]
```

Comment:
Shouldn't this secret already come from the service account mounted to the pod?

Reply:
As I mention in the previous comment the service account used for the k8s plugin would not be used as the SA running the pod, thats why I added this here. But since we will use the SA for running the pod now I will remove this.